calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -142,6 +142,8 @@
 	
 	private Cache cache = null;
 	
+	private String m_calRefList = "";
```

Comment:
What do you think about storing references in an HttpSession object? Would it be a better solution than using the Map? Any other idea?